freqtrade/freqtradebot.py

```diff
@@ -379,9 +380,9 @@ def handle_insufficient_funds(self, trade: Trade):
             except ExchangeError:
                 logger.warning(f"Error updating {order.order_id}.")
 
-#
-# BUY / enter positions / open trades logic and methods
-#
+    #
+    # BUY / enter positions / open trades logic and methods
+    #
```

Comment:
Please remove all unnecessary formatting changes.

they're not relevant to this PR - and formatting in many parts is done intentionally as is.
This does not affect only this segment - but all segments where only whitespace was changed.